common/scripts/report_build_info.sh

```diff
@@ -38,14 +38,17 @@ fi
 
 # security wanted VERSION='unknown'
 VERSION="${BUILD_GIT_REVISION}"
-if [[ -n ${ISTIO_VERSION} ]]; then
-  VERSION="${ISTIO_VERSION}"
+if [[ -n ${TAG_VARIANT} ]]; then
+  VERSION="${TAG_VARIANT}"
 fi
 
 GIT_DESCRIBE_TAG=$(git describe --tags)
+HUB=""
```

Comment:
This means HUB is always `""`. We should instead set `HUB=${HUB:-""}`, so it can be overriden